app/dashboard/views.py

```diff
@@ -133,7 +138,10 @@ def gh_login(request):
 
 def get_interest_modal(request):
 
+    bounty = Bounty.objects.get(pk=request.GET.get("pk"))
```

Comment:
Should we catch if `request.GET.get('pk')` is not present?

Reply:
it should always be present per the JS tho?